app/controllers/articles_controller.rb

```diff
@@ -13,17 +13,25 @@ def new
 
   def send_last_10
     @articles = Article.last(10)
-    UserMailer.last_10_email(current_user,@articles).deliver
+    UserMailer.last_10_email(current_user.email, @articles).deliver_now
     redirect_to root_path
   end
 
+  def send_last_10_by_last_5
+    # def get_content_to_display
+    HardWorker.perform_async(current_user.email)
+    render :processing_email do |page|
```

Comment:
I've never used this sintaxis. But I assume you are returning an html and loading it by ajax in the `remote_function`.

There is I think a better way to do this. The button could be a static button. And by jquery and ajax when you click it, you can make a request to `send_last_10_by_last_5`. This returns an html and by ajax you append it to your page.